bioptim/optimization/optimal_control_program.py

```diff
@@ -140,6 +141,7 @@ def __init__(
         phase_transitions: PhaseTransitionList = None,
         n_threads: int = 1,
         use_sx: bool = False,
+        fatigue_dynamics: FatigueDynamicsList = None,
```

Comment:
Please add a todo to send the fatigue not in OptimalControlProgram but in the DynamicsList